### PR TITLE
feat(LOPS-1103): fix doc and change naming for consitency

### PR DESCRIPTION
**Pull request by FannyGaudin**

### CODE REVIEW

File: src/kili/use_cases/cloud_storage/__init__.py [hunk 1]
```diff
@@ -255,9 +255,10 @@ def update_data_integration(
         status: Optional[DataIntegrationStatus],
     ) -> Dict:
         """Update data integration."""
+        formated_organization_id = organization_id if isinstance(organization_id, str) else ""
```
Comment (Jonas1312) [inline src/kili/use_cases/cloud_storage/__init__.py:258]: should be :

```python
organization_id=OrganizationId(organization_id) if organization_id else None,
```

not: formated_organization_id = organization_id if isinstance(organization_id, str) else ""

File: src/kili/presentation/client/cloud_storage.py [hunk 2]
```diff
@@ -388,16 +388,14 @@ def create_cloud_storage_integration(
...
+        # pylint: disable=line-too-long
         """Create a cloud storage integration.
 
         Args:
-            fields: All the fields to request among the possible fields for the
-            cloud storage integrations.
-                See [the documentation]
-                (https://docs.kili-technology.com/reference/graphql-api#dataintegration)
-                for all possible fields.
+            fields: All the fields to request among the possible fields for the cloud storage integrations.
```
Comment (Jonas1312) [inline src/kili/presentation/client/cloud_storage.py:395]: for the cloud storage integration

File: src/kili/presentation/client/cloud_storage.py [hunk 3]
```diff
@@ -499,10 +497,10 @@ def update_data_integration(
         """
         return CloudStorageUseCases(self.kili_api_gateway).update_data_integration(
             name=name,
-            data_integration_id=DataIntegrationId(data_integration_id),
+            cloud_storage_integration_id=DataIntegrationId(cloud_storage_integration_id),
```
Comment (Jonas1312) [inline src/kili/presentation/client/cloud_storage.py:500]: for use cases, gateway, you can keep the "data integration, data connection"

we only use "cloud storage" for doc and user facing methods

File: src/kili/use_cases/cloud_storage/__init__.py (hunk 1 above)
Comment (FannyGaudin) [reply src/kili/use_cases/cloud_storage/__init__.py:258]: Yes but if I do it like this i have this error :
`Expression of type "tuple[OrganizationId | None]" cannot be assigned to declared type "str | None"`

File: src/kili/presentation/client/cloud_storage.py (hunk 3 above)
Comment (FannyGaudin) [reply src/kili/presentation/client/cloud_storage.py:500]: Yes but in this file cloud_storage_integration_id was used everywhere so we decided to use it as well for keep consistency, is it ok for you ?

File: src/kili/use_cases/cloud_storage/__init__.py (hunk 1 above)
Comment (Jonas1312) [reply src/kili/use_cases/cloud_storage/__init__.py:258]: you  can type the organization_id field of DataIntegrationData  to a Optional[OrganizationId]

Comment (FannyGaudin) [reply src/kili/use_cases/cloud_storage/__init__.py:258]: Oh yes my bad I already did but the formatter didn’t work properly, thanks

File: src/kili/presentation/client/cloud_storage.py (hunk 3 above)
Comment (FannyGaudin) [reply src/kili/presentation/client/cloud_storage.py]: I have update the code for be more clear

File: src/kili/presentation/client/cloud_storage.py (hunk 2 above)
Comment (FannyGaudin) [reply src/kili/presentation/client/cloud_storage.py]: Done

**Review by HNicolas — APPROVED** [review]

**Review by Jonas1312 — APPROVED** [review]

**Pull request merged**